docs/asyncapi.yaml

```diff
@@ -102,22 +102,33 @@ components:
       type: string
       description: The name of the source system that originated the message
       example: "AHWP"
+      minleght: 3
+      maxlenght: 100
+      pattern: "^[a-z0-9-_]+$"
 
     notifytemplateid:
       type: string
       description: Notification template identifier
       example: "f33517ff-2a88-4f6e-b855-c550268ce08a"
+      minleght: 3
+      maxlenght: 100
+      format: uuid
 
     commstype:
       type: string
       description: Type of communication (e.g., email, SMS)
       default: "email"
       example: "email"
+      minleght: 3
```

Comment:
All we accept is `email` currently so would rather that was included as an allowed values array.  Rather than length